tests/test_matrix_utils_replies.py

```
@@ -0,0 +1,722 @@
+
+@pytest.mark.asyncio
+async def test_send_reply_to_meshtastic_with_reply_id():
+    """Test sending a reply to Meshtastic with reply_id."""
+    mock_room_config = {"meshtastic_channel": 0}
+    mock_room = MagicMock()
+    mock_event = MagicMock()
+
+    real_loop = asyncio.get_running_loop()
+
+    class DummyLoop:
+        def __init__(self, loop):
```

Ruff (0.15.9)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)